packages/renderer/src/lib/kube/details/KubePort.svelte

```diff
@@ -75,18 +76,39 @@ async function removePortForward(): Promise<void> {
 <span aria-label="port {port.value}" class="flex gap-x-2 items-center">
   {port.displayValue}
   {#if mapping}
-    <Button title="Open in browser" disabled={loading} icon={faSquareUpRight} on:click={openExternal.bind(undefined)} class="px-1 py-0.5" padding="0">
+    <Button
+      title="Open in browser"
+      disabled={loading}
+      icon={faSquareUpRight}
+      on:click={openExternal.bind(undefined)}
+      class="px-1 py-0.5"
+      padding="0">
       Open
     </Button>
-    <Button title="Remove port forward" disabled={loading} icon={faTrash} on:click={removePortForward.bind(undefined)} class="px-1 py-0.5" padding="0">
+    <Button
+      title="Remove port forward"
+      disabled={loading}
+      icon={faTrash}
+      on:click={removePortForward.bind(undefined)}
+      class="px-1 py-0.5"
+      padding="0">
       Remove
     </Button>
-  {:else}
-    <Button title="Forward port {port.value}" disabled={loading} on:click={onForwardRequest.bind(undefined, port)} class="px-1 py-0.5" padding="0">
+  {:else if (port.protocol ?? 'TCP') === 'TCP'}
+    <Button
+      title="Forward port {port.value}"
+      disabled={loading || (!!port.protocol && port.protocol !== 'TCP')}
```

Comment:
Can the condition `!!port.protocol && port.protocol !== 'TCP'` be false? (we are in a `if !protocol || protocol === 'TCP'` which is the exact opposite)

Reply:
Oh yeah, it should not be here, sorry at first I though about disabling the button, then I thought it would be better to just remove it